src/winter_onboarding_2021/core.clj

```diff
@@ -0,0 +1 @@
+(ns winter-onboarding-2021.core)
```

Comment:
Why is this necessary?

Reply:
I was unable to start repl, because of absence of core file in fms(fleet management service)

Reply:
This file is not necessary. We have change the init-ns in the project clj to user.clj

Reply:
Ok. Shall I make the change in this PR itself?

Reply:
Ok, cool. Will remove it.